docs/src/main/sphinx/admin.md

```diff
@@ -27,6 +27,21 @@ documentation](/connector).
 (admin-event-listeners)=
 ## Event listeners
 
+Event listeners are plugins that allow streaming of query events, such query
+started or query finished to an external system. 
+
+By default, query events are only reflected as internal state changes and as
```

Comment:
I'd like to phrase this something like

> Unrelated to event listeners, the coordinator stores information about recent queries in memory for usage by the web UI. See `properties-query-management`.

Maybe put this after the list of event listeners, since this is separate

Reply:
Updated now.